tests/ssl-opt.sh

```diff
@@ -8260,7 +8285,7 @@ run_test    "DTLS reordering: Buffer out-of-order handshake message on client" \
 
 run_test    "DTLS reordering: Buffer out-of-order handshake message fragment on client" \
             -p "$P_PXY delay_srv=ServerHello" \
-            "$P_SRV mtu=512 dgram_packing=0 cookies=0 dtls=1 debug_level=2 \
+            "$P_SRV mtu=256 dgram_packing=0 cookies=0 dtls=1 debug_level=2 \
```

Comment:
Question: can you explain to me (1) why the MTU has to be changed, and (2) why the new value keeps the test effective (that is, if there was a bug in the feature being tested, would that bug still be found by the test with the new value)? Thanks!

Reply:
This is something that I am not certain myself, with mtu of 512, there wasn't fragmentation seen according to the debug-logs and the test failed, dropping the mtu to 256 seems to be working. Is the handshake-message smaller with the baremetal configuration thus not fragmenting the messages with original mtu?

Reply:
Oh yes, messages are probably smaller as we use smaller certificates. And the message being buffered in this test is precisely the server's Certificate message (which comes right after the ServerHello that's being delayed here). Problem solved.